src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -100,7 +100,7 @@ private static IList<ComponentDescriptor> ResolveComponentMetadata(HttpContext h
                 {
                     throw new InvalidOperationException(
                         $"{nameof(ComponentHub)} doesn't have an associated endpoint. " +
-                        "Use 'app.UseRouting(routes => routes.MapComponentHub<App>(\"app\"))' to register your hub.");
+                        "Use 'app.UsedEndpoints(endpoints => endpoints.MapComponentHub<App>(\"app\"))' to register your hub.");
```

Comment:
Components uses 100% recycled UsedEndpoints.

Reply:
You can clearly see that the new version is 100% green.






(I am proud of that joke)